core/iwasm/common/wasm_runtime_common.c

```diff
@@ -10,6 +10,7 @@
 #include "wasm_native.h"
 #include "wasm_runtime_common.h"
 #include "wasm_memory.h"
+#include "../../shared/mem-alloc/ems/ems_gc_internal.h"
```

Comment:
should it be wrapped with " WASM_ENABLE_GC_PERFORMANCE" ?

Reply:
Could we add API in the mem allocator and call it instead? Note that we had added a similar API before:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/shared/mem-alloc/mem_alloc.h#L48-L49